scripts/check_release_artifacts.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# should be run in CI or manually as:
+# ./check_release_artifacts.sh v2.3.1
+# DOCKER_USER and DOCKER_PASS need to be set for dockerhub checks when run manually
+set -e
+
+if [[ -n "$1" ]]; then
+    release=${1}
+else
+    echo specify release version e.g. ${0} v2.3.0 >&2; exit 1
+fi
+
+failed=false
+if [[ -n $release && $release =~ ^v([0-9]+\.[0-9]+\.[0-9]+(-[a-z0-9]+)*)$ ]]; then
+    version=${BASH_REMATCH[1]}
+else
+    version=$release
+fi
+
+declare -a targets=(ubuntu macos)
+repo_name=aeternity/aeternity
+
+check_release_asset() {
+    curl -fsS -o /dev/null --head "$1" 2>&1
+}
+
+check_github_release_assets() {
+    for target in "${targets[@]}"; do
+        GH_ASSET=https://github.com/$repo_name/releases/download/$release/aeternity-$version-$target-x86_64.tar.gz
```

Comment:
releases.ops.aeternity.com should be checked as well

Reply:
It's checked by `check_s3_artifacts`